engine/access/rpc/client/grpc_secure_client.go

```diff
@@ -0,0 +1,106 @@
+package main
```

Comment:
IIUC we're going to use this with, e.g. a consensus node as the client. Should this rather be in `utils/ rpc/ client/grpc_secure_client.go`somewhere?